### ISSUE TITLE
[Bug] 'RNCCameraRoll' could not be found. Verify that a module by this name is registered in the native binary.

### ISSUE THREAD

**Issue by Jelmerovereem** [ant-design/ant-design-mobile-rn#1315]
### 💬 Before You Start

- [X] I have searched the [issues](https://github.com/ant-design/ant-design-mobile-rn/issues) of this repository and believe that this is not a duplicate.

### 🙋 Description/Step to reproduce

I'm running expo `49.0.13` & react-native `0.72.5`.

1. Install with: `npx expo install @ant-design/react-native`
2. Install peer dependencies: `npx expo install @react-native-camera-roll/camera-roll @react-native-picker/picker @react-native-community/segmented-control @react-native-community/slider react-native-gesture-handler`
3. Run app: `npm run start`. (you have to have at least 1 antd element used in your app)
4. Open expo app on android, then it throws the error:
![Screenshot from 2023-10-13 16 50 20@2x](https://github.com/ant-design/ant-design-mobile-rn/assets/58043913/168e3d1e-ffd8-4608-bf18-d6d61a89f524)
`Invariant Violation: TurboModuleRegistry.getEnforcing(...): 'RNCCameraRoll' could not be found. Verify that a module by this name is registered in the native binary., js engine: hermes`

### 💻 Environment

Android, Expo

### ⚫️ Output of `npx react-native info`

```sh
System:
  OS: Linux 6.2 elementary OS 7.1 Horus 7.1 Horus
  CPU: (20) x64 13th Gen Intel(R) Core(TM) i9-13900H
  Memory: 17.22 GB / 31.13 GB
  Shell:
    version: 5.1.16
    path: /bin/bash
Binaries:
  Node:
    version: 18.18.0
    path: ~/.nvm/versions/node/v18.18.0/bin/node
  Yarn: Not Found
  npm:
    version: 9.8.1
    path: ~/.nvm/versions/node/v18.18.0/bin/npm
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
IDEs:
  Android Studio: Not Found
Languages:
  Java: Not Found
  Ruby:
    version: 3.0.2
    path: /usr/bin/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.5
    wanted: 0.72.5
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Additional comments

What I tried:
- `npx react-native link`
- `npx react-native-asset`

**Comment by krutilins** [comment]
same for me:

```
System:
  OS: macOS 13.5
  CPU: (10) arm64 Apple M1 Pro
  Memory: 106.97 MB / 16.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 18.16.0
    path: ~/.nvm/versions/node/v18.16.0/bin/node
  Yarn:
    version: 3.4.1
    path: ~/.nvm/versions/node/v18.16.0/bin/yarn
  npm:
    version: 9.5.1
    path: ~/.nvm/versions/node/v18.16.0/bin/npm
  Watchman:
    version: 2023.11.13.00
    path: /opt/homebrew/bin/watchman
Managers:
  CocoaPods:
    version: 1.14.3
    path: /Users/sergeykrutilin/.rbenv/shims/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 23.0
      - iOS 17.0
      - macOS 14.0
      - tvOS 17.0
      - watchOS 10.0
  Android SDK: Not Found
IDEs:
  Android Studio: Flamingo 2022.2.1 Flamingo 2022.2.1
  Xcode:
    version: 15.0.1/15A507
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 17.0.6
    path: /usr/bin/javac
  Ruby:
    version: 2.6.10
    path: /Users/sergeykrutilin/.rbenv/shims/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react: Not Found
  react-native: Not Found
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false
```

**Comment by Lemonade1172** [comment]
same question, but expo 49.0.15 & react-native 0.72.6.

**Comment by chenyu1990** [comment]
还没有解决吗？

**Comment by yorickshan** [comment]
same issue

**Comment by chenyu1990** [comment]
expo install

Shan Yinlong ***@***.***>于2023年12月28日 周四15:30写道：

**Comment by 1uokun** [comment]
fixed  in 5.1.0 by: 
> ❗️Delete `ImagePicker` component and remove dependence `@react-native-camera-roll/camera-roll`
